pkg/services/publicdashboards/models/models.go

```diff
@@ -40,20 +40,39 @@ var (
 type ShareType string
 
 type PublicDashboard struct {
-	Uid                  string        `json:"uid" xorm:"pk uid"`
-	DashboardUid         string        `json:"dashboardUid" xorm:"dashboard_uid"`
-	OrgId                int64         `json:"-" xorm:"org_id"` // Don't ever marshal orgId to Json
+	Uid          string    `json:"uid" xorm:"pk uid"`
+	DashboardUid string    `json:"dashboardUid" xorm:"dashboard_uid"`
+	OrgId        int64     `json:"-" xorm:"org_id"` // Don't ever marshal orgId to Json
+	AccessToken  string    `json:"accessToken" xorm:"access_token"`
+	CreatedBy    int64     `json:"createdBy" xorm:"created_by"`
+	UpdatedBy    int64     `json:"updatedBy" xorm:"updated_by"`
+	CreatedAt    time.Time `json:"createdAt" xorm:"created_at"`
+	UpdatedAt    time.Time `json:"updatedAt" xorm:"updated_at"`
+	//config fields
 	TimeSettings         *TimeSettings `json:"timeSettings" xorm:"time_settings"`
-	IsEnabled            bool          `json:"isEnabled" xorm:"is_enabled"`
-	AccessToken          string        `json:"accessToken" xorm:"access_token"`
-	AnnotationsEnabled   bool          `json:"annotationsEnabled" xorm:"annotations_enabled"`
 	TimeSelectionEnabled bool          `json:"timeSelectionEnabled" xorm:"time_selection_enabled"`
+	IsEnabled            bool          `json:"isEnabled" xorm:"is_enabled"`
+	AnnotationsEnabled   bool          `json:"annotationsEnabled,omitempty" xorm:"annotations_enabled"`
 	Share                ShareType     `json:"share" xorm:"share"`
 	Recipients           []EmailDTO    `json:"recipients,omitempty" xorm:"-"`
-	CreatedBy            int64         `json:"createdBy" xorm:"created_by"`
-	UpdatedBy            int64         `json:"updatedBy" xorm:"updated_by"`
-	CreatedAt            time.Time     `json:"createdAt" xorm:"created_at"`
-	UpdatedAt            time.Time     `json:"updatedAt" xorm:"updated_at"`
+}
+
+type PublicDashboardDTO struct {
```

Comment:
I like the idea of having a DTO, with this change maybe we should stop using the entity representation on responses and in that case, `recipients` should be removed from the `PublicDashboard` entity.
Also, we should re-think about which fields we really need, e.g. the `OrgId` is not being used so it shouldn't be present in this DTO. Not sure if all these changes should be in this PR though.

Reply:
I would do this in another PR